modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionFrontendTokenDefinition/testShouldCreateConfigFileWithFrontendTokenDefinitionJson.gradle

```diff
@@ -0,0 +1,25 @@
+import groovy.json.JsonSlurper
+
+task runGradleTest
+
+runGradleTest {
+	dependsOn ":client-extension:any-theme-css-project-create-config-file-frontend-token-definition:build"
+	doLast {
+		Project project = project(":client-extension:any-theme-css-project-create-config-file-frontend-token-definition")
+
+		File buildDir = new File(project.buildDir, "liferay-client-extension-build")
+
+		File configJsonFile = new File(buildDir, "any-theme-css-project-create-config-file-frontend-token-definition.client-extension-config.json")
```

Comment:
usually we write JSON as allCaps unless it is the first word. I think this should be `configJSONFile`. Is that true here too @drewbrokke ?

If so, make sure to change all usages.

Reply:
Unfortunately it's not consistent in workspace, but for `configJSON` it's always capitalized. I'd go with you on this one @ethib137.

Reply:
cool, fix done